app/assets/stylesheets/sass/core.scss

```diff
@@ -2,6 +2,10 @@
 @import "neat/neat";
 @import "base/base";
 
+@import "colors";
```

Comment:
the import order is important. I wanted to overwrite any colors definted in neat or bourban, but set them before anything else.